File: packages/java/src/main/java/com/peruspatial/territorial/HazardForCategory.java
package com.peruspatial.territorial;

import com.peruspatial.data.TerritorialData;

public final class HazardForCategory {
  private HazardForCategory() {}

  public static String hazardForCategory(String category) {
    String found = TerritorialData.CATEGORY_FALLBACK.get(category);
    if (found != null) return found;
    return TerritorialData.DEFAULT_HAZARD;
  }
}

File: packages/rust/src/territorial/hazard_for_category.rs
use crate::data::territorial::{CATEGORY_FALLBACK, HAZARD_DEFAULT};

pub fn hazard_for_category(category: &str) -> String {
    for (key, value) in CATEGORY_FALLBACK {
        if *key == category {
            return value.to_string();
        }
    }
    HAZARD_DEFAULT.to_string()
}
